docker/nginx/default.conf

```diff
@@ -18,6 +18,10 @@ server {
         rewrite ^/module/(.*)/(imagens|scripts|styles)/(.*)$ /intranet/$2/$3 break;
     }
 
+    location ~ /password/(.*)/(styles|scripts|imagens)/(.*) {
```

Comment:
Qual o motivo desta rescrita?

Reply:
Vish @edersoares, desculpa, essa alteração não tem relação com o Docker.

Esse ajuste é para a logo principal ser apresentada na página de redefinição de senha, é um BUG atual no i-Educar.

Poderia juntar o modules e password com o `|` porém não consegui fazer funcionar

Exemplo: `rewrite ^/module|password/`

Dessa forma diminui a quantidade de linhas. Ou rejeitar esse ajuste nesse arquivo que solicito outro PR se for o caso.

Reply:
Ainda não entendi o motivo, acho melhor tratar isolado em outro PR com mais contexto. Por hora isso não sobe.

Reply:
Podemos tratar sim em um PR isolado e com mais contexto.